src/screens/Loading.js

```diff
@@ -63,10 +55,33 @@ export default class Loading extends React.PureComponent {
           })
         ]
       });
-      this.props.navigation.dispatch(tocAction);
-      return;
+    } else {
+      tocActions = firstScreenActions;
     }
-    this.props.navigation.dispatch(firstScreenActions);
+    
+    await loadAccounts()
+    this.props.navigation.dispatch(tocActions);
+  }
+
+  async migrateAccount_v1 () {
+    const oldAccounts_v1 = await loadAccounts(1);
+    const accounts = oldAccounts_v1.map(empty).map(a => ({ ...a, v1recov: true }));
+    accounts.forEach(saveAccount);
+  }
```

Comment:
so this loads v1 accounts from secure storage, resets all the values to something false-y, and adds a new field `v1recov` to set to `true` the saves them back to v2 secureStorage? why does it map `empty`, shouldn't it save it into `v2` storage as is?

Reply:
Just in case you hadn't seen, I copied this code from before.
>  why does it map empty

not sure either.. but my guess is just to make sure we initialize all the new values.

> shouldn't it save it into v2 storage as is?

I can't tell what's the diff between v1 and v2.. so I can't tell what would make sense or not. We would need to test an older version. I'll do that.